CHANGELOG.md

```diff
@@ -52,6 +52,127 @@ pay attention to. Medium impact changes are also worth looking at.
 
 <!--releases-->
 
+## 2024-10-01: 2024q3 release
+
+This release covers everything from 2024-07-01 and has been tested with avr-gcc
+v12.2.0 from Upstream and arm-none-eabi-gcc v12.2.1 from xpack.
+
+Breaking changes:
+
+- Deprecate `modm::allocator::Dynamic` in favor of `std::allocator`.
+
```

Comment:
Another breaking change: https://github.com/modm-io/modm/commit/7a4df093f12f3b53281e6afe3a9568ce5b69d1af

Reply:
Technically ce9c654024c384a76fb99b24fc4634b0dafa8ae7 is also a breaking change since I modified some field names in the config struct. It only impacts SAM users who were providing their own config and the meaning of the parameters is unchanged so there's no risk of a silent break.